README.md

```diff
@@ -35,7 +35,8 @@ It is generally recommended to use the latest point release of the relevant bran
 
 | Elastica branch                                       | ElasticSearch | elasticsearch-php | PHP            |
 | ----------------------------------------------------- | ------------- | ----------------- | -------------- |
-| [7.x](https://github.com/ruflin/Elastica/tree/master) | 7.x           | ^7.0              | ^7.2 \|\| ^8.0 |
+| [8.x](https://github.com/ruflin/Elastica/tree/master) | 8.x           | ^8.4              | >=8.0 <8.2
```

Comment:
The right link to 8.x branch is https://github.com/ruflin/Elastica/tree/8.x

Reply:
@sidz Thank you for your feedback, I made this change based on the history of changes in the README during which I noted that the last branch was systematically positioned on the master branch which itself is an alias which points to the branch of the latest version. I took this to be a convention. I misunderstand ?

Reply:
There was a recent change around 8.x where we renamed `master` to `8.x`. There is no master branch anymore.

Reply:
@ruflin @sidz amended